zipline/src/commonMain/kotlin/app/cash/zipline/internal/bridge/Endpoint.kt

```diff
@@ -112,7 +111,7 @@ class Endpoint internal constructor(
     }
   }
 
-  private val functionsCache = mutableMapOf<String, List<ZiplineFunction<*>>>()
+  private val serviceTypeCache = mutableMapOf<String, RealZiplineServiceType<*>>()
```

Comment:
I’m changing the cache from `List<ZiplineFunction>` to `ZiplineServiceTypes`. Big benefit there is we don’t need to build the HashMap organized by name every single time.